### DISCUSSION TITLE
How to set the picture to directly display the original File Storage picture?

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#8837] [Archive]
![image](https://user-images.githubusercontent.com/8030134/137542175-e2fa2291-c69e-49a9-b73a-af4f6dee1874.png)

How to set the picture to directly display File Storage original pictures instead of generating local pictures?

**Comment by rijkvanzanten** [comment]
There's no way to configure that. The Directus app will display size-appropriate thumbnails in the file library to save on bandwidth and browser memory usage

**Reply by cdwmhcc** [reply]
OK
